modules/kuberay-cluster/variables.tf

```diff
@@ -172,3 +172,15 @@ variable "disable_network_policy" {
   type        = bool
   default     = false
 }
+
+# These default resource quotas are set intentionally high as an example that won't be limiting for most clusters.
+# Consult https://kubernetes.io/docs/concepts/policy/resource-quotas/ for additional quotas that may be set.
+variable "resource_quotas" {
```

Comment:
Resource Quota is deprecated starting in GKE v1.28 right?

Reply:
[citation needed]